src/main/resources/db/migration/V202409231158__create_table_feedback.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE feedback (
+    id SERIAL PRIMARY KEY,
+    employee_id INT NOT NULL,
+    company_id INT NOT NULL,
+    date_of_feedback DATE DEFAULT current_date,
+    anonymity VARCHAR(255) NOT NULL,
+    reviewed BOOLEAN NOT NULL DEFAULT false,
+    feedback VARCHAR(255) NOT NULL,
```

Comment:
This means that the feedback can't be longer than 255 characters. Same for response.
Is this what you meant it to be?

Reply:
yes

Reply:
make sure it matches the spec

Reply:
the spec says to implement validation for feedback and gives as an example to enforce the minimum length of a feedback. 
So i thought other validations is for my decision. do you think I shouldn't put a maximum length?

Reply:
it's ok as long as you can explain it and it makes sense. If this is the length just because this is the default that is usually used, then this decision is not good.
If you think that this limit is enough, you can leave it

Reply:
I'll maybe change it to a different number, usually websites let you write a maximum of round numbers